.github/workflows/releaseWithCoreBundle.yml

```diff
@@ -0,0 +1,16 @@
+name: publish source-deploy-retrieve-bundle
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Set the branch to use for release'
+        type: string
+        required: false
+        default: 'main'
+
+jobs:
+  call-release-workflow:
+    uses: forcedotcom/bundle-publish-scripts/.github/workflows/releaseWithCoreBundle.yml@main
```

Comment:
can I introduce you to fs.readdir with recursive:true instead of writing yet another one of these?  https://github.com/forcedotcom/bundle-publish-scripts/blob/4c0c4a4faf393f7c585c0bf8da153592ecbbee98/scripts/updateForCoreBundle.js#L79.

There's also a `withFileTypes` option if you want to filter out dirs or pass a filter function.

Reply:
Thanks I just had a quick try and the new function only updates the first file. Not sure what went wrong, and I will try to dig into it later